python/py-jupyterlab/Portfile

```diff
@@ -4,14 +4,16 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-jupyterlab
-version             3.0.13
+version             4.0.5
 revision            0
 categories-append   devel
 license             BSD
 supported_archs     noarch
 platforms           {darwin any}
 
-python.versions     37 38 39 310 311
+python.versions     38 39 310 311
```

Comment:
did you verify to make sure that the PY37 subport does not have any dependents anymore? If there are dependents, you will have to pin the PY37 subport to its latest supported version.